### PR TITLE
 Add conditional card for Lovelace

### PR DESCRIPTION
**Pull request by c727**
fix: https://github.com/home-assistant/ui-schema/issues/134

```yaml
    cards:
      - type: entities
        entities:
          - light.bed_light
          - switch.decorative_lights          
      - type: conditional
        conditions:
          - entity: light.bed_light
            state: "on"
          - entity: switch.decorative_lights
            state_not: "off"
        card:
          type: entities
          entities:
            - device_tracker.demo_paulus
            - cover.kitchen_window
            - group.kitchen
            - lock.kitchen_door
            - light.bed_light
```

### CODE REVIEW

**Review by andrey-git — COMMENTED** [review]
Should the card size return 0 in hidden mode?

File: src/panels/lovelace/cards/hui-conditional-card.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    }
+
+    const element = createCardElement(config.card);
+    element.hass = this.hass;
+    root.appendChild(element);
+  }
+
+  getCardSize() {
+    const el = this.$.card.lastChild;
+    return el && this._visible ? computeCardSize(el) : 0;
```
Comment (balloob) [inline src/panels/lovelace/cards/hui-conditional-card.js]: The problem with this is that it becomes unpredictable how your UI will look. For it to look a certain way after a refresh,a state needs to be on now. I think that we should aim for cardSize to be static

**Review by balloob — COMMENTED** [review]
Demo entry?

File: src/panels/lovelace/cards/hui-conditional-card.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      root.removeChild(root.lastChild);
+    }
+
+    const element = createCardElement(config.card);
+    element.hass = this.hass;
+    root.appendChild(element);
+  }
+
+  getCardSize() {
+    return computeCardSize(this.$.card.lastChild);
```
Comment (andrey-git) [inline src/panels/lovelace/cards/hui-conditional-card.js]: this.$.card.lastChild might be undefined.

**Review by balloob — APPROVED** [review]

**Pull request merged**